src/features/AdminPage/CategoriesPage/CategoriesPage/CategoryAdminModal.component.tsx

```
@@ -26,8 +26,15 @@ import base64ToUrl from '@/app/common/utils/base64ToUrl.utility';
 import normaliseWhitespaces from '@/app/common/utils/normaliseWhitespaces';
 import uniquenessValidator from '@/app/common/utils/uniquenessValidator';
 
-import POPOVER_CONTENT from '../../JobsPage/JobsModal/constants/popoverContent';
 import PreviewFileModal from '../../NewStreetcode/MainBlock/PreviewFileModal/PreviewFileModal.component';
+import uniquenessValidator from '@/app/common/utils/uniquenessValidator';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'uniquenessValidator'. Consider to delete it or rename it.

'uniquenessValidator' is defined here:


(lint/suspicious/noRedeclare)